packages/web-pkg/src/components/SideBar/Spaces/Details/SpaceDetails.vue

```diff
@@ -52,12 +52,16 @@
       </tr>
       <tr v-if="resource.description">
         <th scope="col" class="oc-pr-s oc-font-semibold" v-text="$gettext('Subtitle')" />
-        <td v-text="resource.description" />
+        <td v-oc-browser-translate-off v-text="resource.description" />
       </tr>
       <tr>
         <th scope="col" class="oc-pr-s oc-font-semibold" v-text="$gettext('Manager')" />
         <td>
-          <span v-text="ownerUsernames" />
+          <span v-for="(owner, index) in owners" :key="index">
+            <span v-text="owner.displayName" />
```

Comment:
this one needs the `v-oc-browser-translate-off` directive as well, no?

Reply:
🙈